acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/signer/EthSignerProcessRunner.java

```diff
@@ -140,11 +135,11 @@ public void start(final String processName) {
     params.add(String.valueOf(signerHttpRpcPort));
     params.add("--chain-id");
     params.add(chainId);
-
     if (useDynamicPortAllocation) {
       params.add("--data-directory");
       params.add(dataDirectory.toAbsolutePath().toString());
     }
+    params.addAll(signerConfig.transactionSignerParamsSupplier().params());
```

Comment:
There must be a short / more sensible method name for this (`param` appearing twice seems odd)

Reply:
done